lambdas/ssm-parameters/src/ssm-parameters-handler.ts

```diff
@@ -0,0 +1,38 @@
+import { LambdaInterface } from "@aws-lambda-powertools/commons";
+import { getParametersByName } from "@aws-lambda-powertools/parameters/ssm";
+import { Parameter } from "@aws-sdk/client-ssm";
+
+const PARAMETER_TTL =
+  parseNumber(process.env.POWERTOOLS_PARAMETERS_MAX_AGE) ?? 300;
+
+function parseNumber(value?: string) {
+  return parseInt(value ?? "", 10) || undefined;
+}
+export class SsmParametersHandler implements LambdaInterface {
+  public async handler(
+    event: string[],
+    _context: unknown
+  ): Promise<Parameter[]> {
+    if (!Array.isArray(event)) {
+      throw new Error(`Input must be string array`);
+    }
+
+    const { _errors: errors, ...parameters } =
+      await getParametersByName<string>(
+        Object.fromEntries(event.map((parameter) => [parameter, {}])),
+        { maxAge: PARAMETER_TTL, throwOnError: false }
+      );
+
+    if (errors?.length) {
+      throw new Error(`Couldn't retrieve SSM parameters: ${errors.join(", ")}`);
```

Comment:
Can you change this error message and to include context? In this situation, it's not that it "couldn't retrieve" but it's actually a parameter does not exist error. 

It's better to be concise on why it's erroring otherwise it'll be harder to debug because a developer might not know why it cannot retrieve.